content/blog/automate-boot-from-san-target-configurations-using-redfish.md

```diff
@@ -0,0 +1,323 @@
+
+There are several use-cases of infrastructure automation that exist, and in this blog post we will address the aspect of configuration of OS boot for a server from a Storage Area Network (SAN) based storage array.  Specifically, we will discuss the   procedure for configuring an HPE ProLiant DL Gen10, Gen10 plus and Gen11 servers to boot from a SAN target Logical Unit Number (LUN) using the Redfish® standard DMTF API. 
+
+**Note**:
+
+* The steps outlined in this document were tested on HPE DL 380 Gen10 and Gen10 plus servers with Emulex based SN1610E FibreChannel (FC) cards, HPE iLO firmware 2.81 and Service Pack for ProLiant (SPP) available as of March 2023.
+* QLogic SN1610Q cards do not support the steps outlined in this document (as of April 2023) However, future firmware releases might enable this Redfish API based configuration method.
+
+# Pre-requisite information
+
+Before we dive into the steps for performing the automation, you will need to gather a few pieces of information, such as
+
+* HPE iLO Management port ip address and credentials
+* Storage target World Wide Number (WWN) and LUN id.  A storage administrator can typically provide you with this information.
+
+Download and install the [HPE RESTful Interface Tool](https://www.hpe.com/info/resttool)  (version 4.1.0.0 or later) on the system you will initiate the automation actions from.
+Once the RESTful Interface tool (referred to as iLOrest in the remainder of the document), you can review the current boot target and their order of execution by following the steps below
+
+`ilorest login <ilo-ip> -u <ilo-user> -p password`
+
+`ilorest bootorder`
+
+Refer to the [iLOrest documentation](https://hewlettpackard.github.io/python-redfish-utility/#bootorder-command) for more information on this command.
+
+# Configuration Steps
+
+Now that you have gathered the pre-requisite information and installed the ilorest tool, let’s discuss the detailed steps for implementing the automation.
+
+## Step 1: Ensure that the `FCScanPolicy` value is set to ‘AllTargets’
+
+```markdown
+ilorest  get FCScanPolicy --selector Bios
+```
+
+Expected output:
+
+```markdown
+FCScanPolicy=AllTargets
+```
+
+If the value is not set to AllTargets, you can change it using the following:
+
+```markdown
+ilorest set "FCScanPolicy"="AllTargets" --selector Bios
+
+ilorest commit
+
+ilorest reboot
+```
+
+**Note**:
+
+A﻿fter restarting the server, remember to logout/login again to the server using ilorest tool to retrieve the updated settings from the server directly, rather than the cache.  You can also update the ilorest cache contents with the following command
```

Comment:
Hummm, there is no need to "ilorest logout/login" after server restart. Remember that iLOrest logs into the iLO, NOT the server. The only thing to do is to keep iLOrest logged in the iLO and refresh the Bios data type cache. 

Please, rephrase with:

After restarting the server, you need to update the iLOrest cache related to the Bios resources with the following command:

Reply:
good point, i have made the change